### PR TITLE
[SW-225] Page transitions tweaks

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/styles/global.scss [hunk 1]
```diff
@@ -1,17 +1,46 @@
...
+
+  &.color-bg-3 {
+    background-color: $color-bg-3;
+  }
+
+  @include mixins.desktop {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```
Comment (pstachel) [inline src/styles/global.scss:34]: You could fit it in one line by using flex mixin

**Review by pstachel — CHANGES_REQUESTED** [review]

File: src/styles/global.scss (hunk 1 above)
Comment (peronczyk) [reply src/styles/global.scss:34]: I know that having less code usually is a better idea. But SCSS mixins after all are still transpiled into regular CSS rules. So there's no benefit in speed. And in my opinion makes the code harder to read.

**Comment by CedrikNikita** [comment]
Account card widening

The page is visually changing on this transaction.

**Review by pstachel — APPROVED** [review]

File: src/popup/router/pages/Dashboard.vue [hunk 2]
```diff
@@ -1,98 +1,95 @@
...
+      >
+        <template #icon>
+          <ArrowReceiveIcon />
+        </template>
+      </Card>
+      <Card
+        :title="$t('dashboard.send-card.title')"
+        :description="$t('dashboard.send-card.description')"
+        clickable
+        @click="openTransferSendModal()"
```
Comment (konradodo) [inline src/popup/router/pages/Dashboard.vue:19]: openTransferSendModal

File: src/popup/router/pages/Dashboard.vue [hunk 3]
```diff
@@ -1,98 +1,95 @@
...
-      </CardRow>
-    </Plate>
+  <div class="dashboard">
+    <AccountSwitcher :notification="!backedUpSeed" />
+    <CardRow class="first-card-wrapper">
+      <Card
+        :title="$t('dashboard.receive-card.title')"
+        :description="$t('dashboard.receive-card.description')"
+        clickable
+        @click="openTransferReceiveModal()"
```
Comment (konradodo) [inline src/popup/router/pages/Dashboard.vue:9]: @click="openTransferReceiveModal"

**Review by konradodo — APPROVED** [review]

File: src/popup/router/pages/Dashboard.vue (hunk 3 above)
Comment (peronczyk) [reply src/popup/router/pages/Dashboard.vue]: Talked about it on priv. I'm leaving it as it is.

File: src/popup/router/pages/Dashboard.vue (hunk 2 above)
Comment (peronczyk) [reply src/popup/router/pages/Dashboard.vue]: Talked about it on priv. I'm leaving it as it is.

**Pull request merged**